### PR TITLE
Revert version upgrades

### PR DESCRIPTION
**Pull request by bmuskalla**
Unblock build pipeline by using known versions. Will further investigate what it takes to use the newer versions.

### CODE REVIEW

**Comment by bmuskalla** [comment]
@bot-gradle test QF

File: subprojects/build-init/src/main/resources/org/gradle/buildinit/tasks/templates/library-versions.properties [hunk 1]
```diff
@@ -4,8 +4,8 @@ groovy=2.5.12
 guava=29.0-jre
 junit-jupiter=5.6.2
 junit=4.13
-kotlin=1.4-M2-mt
```
Comment (eskatos) [inline subprojects/build-init/src/main/resources/org/gradle/buildinit/tasks/templates/library-versions.properties:7]: Ha! that won't fly. The automated process should be selecting stable versions only I suppose.

**Comment by ljacomet** [comment]
Merging to unblock `master`

File: subprojects/build-init/src/main/resources/org/gradle/buildinit/tasks/templates/library-versions.properties (hunk 1 above)
Comment (bmuskalla) [reply subprojects/build-init/src/main/resources/org/gradle/buildinit/tasks/templates/library-versions.properties:7]: Followed up: https://github.com/gradle/gradle/pull/13648

**Pull request merged**